Sources/UserModuleKit/Interfaces/Profile+Interface.swift

```diff
@@ -7,7 +7,7 @@
 
 import FeatherModuleKit
 
-public protocol UserProfileInterface {
+public protocol UserProfileInterface: Sendable {
 
     func get() async throws -> User.Account.Detail
```

Comment:
It should be renamed to "require" because of Detail return is not optional (not Detail?)